overlord/ifacestate/handlers.go

```diff
@@ -458,13 +444,9 @@ func (m *InterfaceManager) doDisconnect(task *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
-	affectedConns, err := m.repo.ResolveDisconnect(plugRef.Snap, plugRef.Name, slotRef.Snap, slotRef.Name)
```

Comment:
Very good point. I think we can either keep this logic here for such cases, or have a patch to convert old tasks to new approach. Let's discuss in HO/forum.